src/pages/AuthPage.tsx

```diff
@@ -23,17 +26,32 @@ export const AuthPage = () => {
   return (
     <IonPage>
       <IonContent>
-        <form
+
+      <div className="log">
+          
+
+          {/* <IonIcon name="logo-javascript"></IonIcon> */}
+          {/* <IonIcon icon={logoJavascript} size="large"></IonIcon> */}
+          
+              <div className="logoIMG">
+                <img src="/public/assets/ui/logo.svg"  />
```

Comment:
There needs to be an `alt` prop. 

Installing the `estlint` VS Code plugin would allow you to catch this in your code editor (aka IDE).

Reply:
Fixed in my latest commit